packages/kmod-6.1-nvidia-r570/nvidia-tesla-build-config.toml.in

```diff
@@ -0,0 +1,18 @@
+[nvidia-tesla]
```

Comment:
You must add the previously excluded modules to this file, otherwise they won't be linked at runtime:

- `nvidia-peermem`
- `nvidia-drm`